### PR TITLE
Change font size of event info

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/scenes/Event/components/EventInfo/styles.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const styles = (theme) => ({
     textOverflow: 'ellipsis',
     whiteSpace: 'nowrap',
     overflow: 'hidden',
-    paddingLeft: theme.padding.lg.px,
+    paddingLeft: theme.padding.md.px,
+    fontSize: theme.typography.fontSize,
```
Comment (liviavinci) [inline src/scenes/Event/components/EventInfo/styles.js]: why is this necessary? theme.typography.fontSize is just the default font size right?

Comment (dwalintukan) [reply src/scenes/Event/components/EventInfo/styles.js]: the Result Setter text is being ellipsized. i changed the font size because i wanted it black color still, just smaller text size. suggestions?

Comment (liviavinci) [reply src/scenes/Event/components/EventInfo/styles.js]: oh i did that ellipsis thing on purpose but now as i think of it, it doesn't make much sense. Can you remove textOverflow: 'ellipsis', whiteSpace: 'nowrap',overflow: 'hidden',?

File: src/scenes/Event/components/EventInfo/styles.js [hunk 2]
```diff
@@ -10,7 +10,8 @@ const styles = (theme) => ({
     textOverflow: 'ellipsis',
```
Comment (liviavinci) [inline src/scenes/Event/components/EventInfo/styles.js]: remove

Comment (dwalintukan) [reply src/scenes/Event/components/EventInfo/styles.js]: wont this just cutoff the end of the text and some of the letters will still be missing?

Comment (liviavinci) [reply src/scenes/Event/components/EventInfo/styles.js]: well if we remove textOverflow: 'ellipsis', whiteSpace: 'nowrap',overflow: 'hidden', and add "word-wrap:break-word;". If the text is more than the div width, it will just be two lines

Comment (dwalintukan) [reply src/scenes/Event/components/EventInfo/styles.js]: i'm not sure if we should show a qtum address on two lines. think that feels kind of confusing for the user. can we adjust the grid of the main content to give the side content more room?

Comment (liviavinci) [reply src/scenes/Event/components/EventInfo/styles.js]: but the thing is, no matter how you adjust that, if the user drags the window into a smaller size, you'll have the same problem.

Comment (dwalintukan) [reply src/scenes/Event/components/EventInfo/styles.js]: true, change made.

File: src/scenes/Event/components/EventInfo/styles.js (hunk 1 above)
Comment (dwalintukan) [reply src/scenes/Event/components/EventInfo/styles.js]: done. are you ok with the text size change or you think it should be big?

File: src/scenes/Event/components/EventInfo/styles.js [hunk 3]
```diff
@@ -7,10 +7,9 @@ const styles = (theme) => ({
   },
   eventInfo: {
     marginTop: theme.padding.unit.px,
-    textOverflow: 'ellipsis',
-    whiteSpace: 'nowrap',
-    overflow: 'hidden',
-    paddingLeft: theme.padding.lg.px,
+    'word-wrap': 'break-word',
```
Comment (liviavinci) [inline src/scenes/Event/components/EventInfo/styles.js]: wordWrap: 'break-word',

Comment (dwalintukan) [reply src/scenes/Event/components/EventInfo/styles.js]: fixed

File: src/scenes/Event/components/EventInfo/styles.js (hunk 1 above)
Comment (liviavinci) [reply src/scenes/Event/components/EventInfo/styles.js]: I'm ok with the smaller font size since it would be nice to make the address in one line on most of the laptops
